dataloader.go

```diff
@@ -363,30 +384,27 @@ func (l *Loader) batch() {
 	}
 }
 
-// wait the appropriate amount of time for next batch
-func (l *Loader) sleeper() {
+// wait the appropriate amount of time for the provided batcher
+func (l *Loader) sleeper(b *batcher, close chan struct{}) {
```

Comment:
I usually use `stop` for this kind of channels, not sure why this time I've used `close`... :disappointed:

looks like it's not reserved as a variabile name but only as a function: https://golang.org/ref/spec#Keywords https://golang.org/ref/spec#Predeclared_identifiers